app/components/transfer/TransferMnemonicPage.scss

```diff
@@ -58,6 +58,10 @@
 }
 
 :global(.YoroiModern) .component {
+  .inputWrapper fieldset {
+    background-color: #ffffff !important;
```

Comment:
please avoid hardcoded colors like this when possible. You can see in the `app/themes/prebuilt`  folder the CSS variables used to map names to colors.

Reply:
Yes, my bad. Anyway, now I'm fixing it with new feedback from Marta and adding a new color variable to the theme since it's not an existing color.